src/components/ActivityCard.js

```diff
@@ -53,7 +47,7 @@ const WrapperCard = styled.section`
 `;
 
 const LinkStyling = styled(Link)`
-  padding: 8px 8px 0;
+  padding: 8px 8px 5px;
   font-size: 18px;
```

Comment:
you could move your font-size to global styles for all links...just check I think it´s an a tag but not 100% sure...but it´s not necessary ;)

Reply:
Uhh thank you, i thought i took it out everywhere as i actually put it into global styles😊